server/mdm/apple/vpp/api.go

```diff
@@ -201,7 +201,14 @@ func do[T any](req *http.Request, token string, dest *T) error {
 	}
 
 	if resp.StatusCode != http.StatusOK {
-		return fmt.Errorf("calling Apple VPP endpoint failed with status %d", resp.StatusCode)
+		// use an io.LimitedReader to read only a limited amount of the
+		// response body for logging purposes
```

Comment:
Oh sorry, I didn't realize you already had the body read in full, in that case you could just check the len and limit it to 1024 bytes without using a reader. No big deal either way, though.

Reply:
I was wondering about that! sounds good! changes pushed

Reply:
LGTM, you should remove the comment above though ^

Reply:
I'm dumb, thanks!